File: Python/tests/test_rdt_happiness.py
import pytest

from hbayesdm.models import rdt_happiness


def test_rdt_happiness():
    _ = rdt_happiness(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_rdt_happiness.R
context("Test rdt_happiness")
library(hBayesDM)

test_that("Test rdt_happiness", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(rdt_happiness(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
